kitsune/landings/jinja2/landings/includes/register_contributor.html

```diff
@@ -24,10 +24,10 @@ <h3 class="card--title">
         <button id="sign-up-contributor" class="sumo-button primary-button button-lg" data-event-category="Sign Me Up Click" data-event-action="Logged In">{{ _('Sign me up') }} &raquo;</button>
       </form>
     {% else %}
-      <a id="sign-up-contributor" class="sumo-button primary-button" href="{{ url('users.fxa_authentication_init') }}?is_contributor=True" data-event-category="Sign Me Up Click" data-event-action="Logged In">
+      <a id="sign-up-contributor" class="sumo-button primary-button" href="/{{ url('users.fxa_authentication_init') }}?next={{ next_url }}&is_contributor=True" data-event-category="Sign Me Up Click" data-event-action="Logged In">
```

Comment:
`url()` returns a path with a slash at the start, so at the moment this is a link to `//foobar`

Reply:
Sure, will do that.